libs/service-portal/licenses/src/screens/components/LicenseCards/LicenseCards.tsx

```diff
@@ -284,7 +285,9 @@ const LicenseCards = () => {
             <dt>Staða</dt>
             <dd>{license.license.status}</dd>
             <dt>pkpass?</dt>
-            <dd>{license.license.pkpass}</dd>
+            <dd>{license.license.pkpass ? 'Já' : 'Nei'}</dd>
```

Comment:
Is it ok to have this not translated here?

Reply:
yes, it's just a dummy ui for testing